src/components/stepper/StepTwo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import { FormControl} from "@mui/material";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Radio from "@mui/material/Radio";
+
+
+
+const StepTwo = () => {
+    return (
+        <React.Fragment>
+            <Box sx={{borderBottom: "solid",  borderTop: "solid",borderWidth: 2, borderColor: "primary.main", p:2}} >
+                <FormControl >
+                    <RadioGroup  >
+                        <FormControlLabel value="marketingEmail" control={<Radio />} label="Marketing Email" />
```

Comment:
You should map an array instead of these values and labels instead of hardcoding all of the `<FormControlLabel>`s